src/lib/qboard.ts

```diff
@@ -214,6 +219,21 @@ export default class QBoard {
     await this.history.add([this.currentObject]);
   };
 
+  dragEnter = (e: DragEvent): void =>
+    this.dropArea.classList.add("file-drop-active");
+
+  dragOver = this.dragEnter;
```

Comment:
why have dragenter in the first place

Reply:
Not particularly needed in the final code. I was experimenting between the two as events and so made functions for each event. I still think it's worth comparing these two events (dragOver is called many times while dragEnter is called once) but this doesn't need to be in the code.

Let me know if I should remove it

Reply:
please do

Reply:
dragEnter is currently used and dragOver is currently useless. I will delete dragEnter

Reply:
Keeping dragEnter and removing dragOver since the dragover event is listened to in the current code